provision/acc_provision/apic_provision.py

```diff
@@ -3456,5 +3412,116 @@ def cloudfoundry_tn(self):
+                                        "ip",
+                                    ),
+                                    (
+                                        "prot",
+                                        "tcp",
+                                    ),
+                                    (
+                                        "dFromPort",
+                                        "2379",
+                                    ),
+                                    (
+                                        "dToPort",
+                                        "2379",
+                                    ),
+                                    (
+                                        "stateful",
+                                        "no",
+                                    ),
+                                    (
+                                        "tcpRules",
+                                        "",
+                                    ),
+                                ]
+                            ),
+                        )
+                    ]
+                ),
+            )
+        ]
+    )
+
+    data['fvTenant']['children'][7]['vzFilter']['children'].append(os_extra_port)
+
+
+def open_port_for_dockerucp(data):
+
+    single_ports = ["443", "12376", "10250", "12376", "10250"]
+    range_ports = (12378, 12389)
+    ports = [port for port in single_ports]
+    ports = ports + [str(i) for i in range(range_ports[0], range_ports[1])]
+
+    for port in ports:
+        filter_name = "dockerucp_port_" + port
+        dockerucp_extra_port = collections.OrderedDict(
+            [
+                (
+                    "vzEntry",
+                    collections.OrderedDict(
+                        [
+                            (
+                                "attributes",
+                                collections.OrderedDict(
+                                    [
+                                        (
+                                            "name",
+                                            filter_name,
+                                        ),
+                                        (
+                                            "etherT",
+                                            "ip",
+                                        ),
+                                        (
+                                            "prot",
+                                            "tcp",
+                                        ),
+                                        (
+                                            "dFromPort",
+                                            port,
+                                        ),
+                                        (
+                                            "dToPort",
+                                            port,
+                                        ),
+                                        (
```

Comment:
@gaurav-dalvi what I meant was the dFromPort and dToPort should be the range. This way, the number of filter entries in ACI will be smaller.

Reply:
@jojimt : Got it. Please take a look now.